impersonation-filter/src/test/scala/org/openrepose/filters/impersonation/IdentityResponses.scala

```diff
@@ -0,0 +1,321 @@
+package org.openrepose.filters
+
+
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+
+trait IdentityResponses {
+
+  protected final val VALID_TOKEN = "validToken"
+  protected final val VALID_USER = "validUser"
+
+  def tokenDateFormat(dateTime:DateTime): String = {
+    ISODateTimeFormat.dateTime().print(dateTime)
+  }
+
+  def adminAuthenticationTokenResponse(token: String = "glibglob",
+                                       expires: DateTime = DateTime.now()): String = {
+    val formattedTime = tokenDateFormat(expires)
+
+    s"""
```

Comment:
But then wouldn't I have to parse the file to add my dynamic data?  What's the advantage of that over this?